proxy/src/telemetry/metrics/latency.rs

```diff
@@ -174,4 +183,14 @@ impl Into<u32> for Latency {
     fn into(self) -> u32 {
         self.0
     }
-}
\ No newline at end of file
+}
+
+impl fmt::Display for Latency {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if self.0 == u32::MAX {
+            write!(f, "+Inf")
+        } else {
+            write!(f, "{}", (self.0 / MS_TO_TENTHS_OF_MS) as f64)
```

Comment:
Casting the division to float after the operation won't show any of decimal portion, as the division will already be u32 division. Was the goal to show the decimal points?